### PR TITLE
Use HTTPS where possible

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3387,18 +3387,18 @@
         <dt>[[!ECMAScript]]</dt>
         <dd>
           <ul>
-            <li><a href="https://people.mozilla.org/~jorendorff/es6-draft.html#sec-array-objects"
+            <li><a href="http://ecma-international.org/ecma-262/6.0/#sec-array-objects"
```
Comment (jyasskin) [inline index.html]: This change doesn't match the title of your pull request, although it might be the right thing to do.

Comment (beaufortfrancois) [inline index.html]: I think that's why he added "where possible" in the Commit Message. We may want to merge this PR now. WDYT?

Comment (mathiasbynens) [inline index.html]: Yeah that’s why. HTTPS > HTTP, but canonical spec links > unofficial draft snapshots.

Comment (beaufortfrancois) [inline index.html]: @jyasskin (gentle ping)

Comment (jyasskin) [inline index.html]: Yeah, sorry for the delay. It's more that we should commit this as a separate commit titled "Switch links to the final ECMAScript 6 spec." I'll split it now.

**Comment by jyasskin** [comment]
Merged in 62914209f9d201365c4434eea7e4ab10647ffb57 and 4693017e653a7e5fa059fc1433cab4d3cf7b8d29. Thanks!

**Pull request closed without merging**